lib/main_micro_alp.pm

```diff
@@ -66,55 +66,62 @@ sub load_boot_from_dvd_tests {
     }
 }
 
+sub load_10GB_installation_tests {
+    # boo#load_installation_tests
```

Comment:
Might as well fix the broken bugref - if you can find the right one?

Reply:
found it https://bugzilla.opensuse.org/show_bug.cgi?id=1099762